packages/graphqlgen-templates/typescript-yoga/src/resolvers/Mutation.ts

```diff
@@ -1,7 +1,7 @@
 import { MutationResolvers } from '../generated/graphqlgen'
 
 export const Mutation: MutationResolvers.Type = {
-  createUser: (parent, { name }, ctx) => {
+  createUser: (parent, { name = null }, ctx) => {
```

Comment:
Not needed because the schema already has this (yup it’s a feature in the graphql spec).